<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widgetWinTitle" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>719</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="labelWinTitle">
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>与 XXX 聊天中</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetFileBoard" native="true">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>271</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: none;
	border-radius:0px;
	background-color: #05CE26;
	color: #FFFFFF;
}

QProgressBar {
	border: 1px solid grey;
    border-radius: 0px;
	color: #FFFFFF;
}

QProgressBar::chunk {
	background-color: #05B8CC;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,1">
    <item>
     <widget class="QWidget" name="widgetFileInfo" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>文件传输：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>总大小:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditTotalSize">
             <property name="text">
              <string>0 KB</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>已传输：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditCurrSize">
             <property name="text">
              <string>0 KB</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>速率：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditRate">
             <property name="text">
              <string>10 M/S</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTableView" name="tableViewGroups"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnWinClose">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>0</y>
     <width>33</width>
     <height>29</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/resource/common/ic_close_white.png</normaloff>:/resource/common/ic_close_white.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWinMin">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>0</y>
     <width>33</width>
     <height>29</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/resource/common/ic_min.png</normaloff>:/resource/common/ic_min.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelWinIcon">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/resource/images/robot.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/reses/res/backgroud/bac_cat3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widgetMsg" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>399</y>
     <width>641</width>
     <height>151</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>100</height>
    </size>
   </property>
   <widget class="QPushButton" name="btnAction">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendMsg">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>发送聊天消息</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>发送(&amp;S)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMsg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>100</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>108</y>
      <width>132</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>字符限制: 0 / 255</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>关闭当前窗口</string>
    </property>
    <property name="text">
     <string>关闭(&amp;C)</string>
    </property>
   </widget>
  </widget>
  <widget class="BubbleList" name="widgetBubble" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>51</y>
     <width>641</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widgetSetting" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>634</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolButton" name="toolButton_2">
      <property name="toolTip">
       <string>修改聊天字体</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/resource/common/ic_font.png</normaloff>:/resource/common/ic_font.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_3">
      <property name="toolTip">
       <string>插入表情</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/resource/common/ic_smile.png</normaloff>:/resource/common/ic_smile.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_4">
      <property name="toolTip">
       <string>设置字体颜色</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/resource/common/ic_color.png</normaloff>:/resource/common/ic_color.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="btnSendFile">
      <property name="toolTip">
       <string>发送文件</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/resource/common/ic_file.png</normaloff>:/resource/common/ic_file.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_7">
      <property name="toolTip">
       <string>发送图片</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/resource/common/ic_picture.png</normaloff>:/resource/common/ic_picture.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_6">
      <property name="toolTip">
       <string>截屏</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/resource/common/ic_cut.png</normaloff>:/resource/common/ic_cut.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>396</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton">
      <property name="toolTip">
       <string>聊天记录</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/resource/common/ic_notes.png</normaloff>:/resource/common/ic_notes.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label_6</zorder>
  <zorder>widgetMsg</zorder>
  <zorder>widgetBubble</zorder>
  <zorder>widgetSetting</zorder>
  <zorder>widgetWinTitle</zorder>
  <zorder>widgetFileBoard</zorder>
  <zorder>btnWinClose</zorder>
  <zorder>btnWinMin</zorder>
  <zorder>labelWinIcon</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BubbleList</class>
   <extends>QWidget</extends>
   <header location="global">bubblechating.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
